src/OpenTelemetry.Api/Trace/TelemetrySpan.cs

```diff
@@ -103,6 +103,7 @@ public TelemetrySpan UpdateName(string name)
         /// <param name="key">Attribute key.</param>
         /// <param name="value">Attribute value. The value may be an <see cref="IEnumerable"/> of primitive types. An enumeration may be iterated multiple times.</param>
         /// <returns>The <see cref="TelemetrySpan"/> instance for chaining.</returns>
+        /// <remarks>More types for value will be supported in the next release. (bool, int etc.)</remarks>
```

Comment:
I think the Activity API just has string & object (boxing of value types, ug). You thinking we'll just add helpers that call the add accepting object?

Reply:
`SetTag(string, object)`  is coming in preview8, yes.